documentation/kernel-semantics.tex

```diff
@@ -1954,7 +1956,7 @@ \subsubsection{Direct Property Extraction}
         \cesktranswheresplit*%
         {\contconf{\DPropertySetVK}{\val_1}}
         {\execconf{\stmt}{\env}{\lbls}{\clbls}{(\DirectPropertySet{\val_0}{\membermeta}{\val_1}) :: \strace}{\handler}{\cex,\,\cstrace}{\econt}{\emptyset}}
-        {\parbox{10cm}{where:\\
+        {\parbox{10cm}{where\\
```

Comment:
I would also remove \\ after "where" here and in other places and also aligned the lines at "=" sign. See how it's done for the transition rule for field initializers (it's not in a figure, and you may find it using the search phrase "be a non-redirecting constructor" that immediately precedes the transition rule; in the current "master" it starts on the line 2335).